src/dispatch/plugins/dispatch_pagerduty/service.py

```diff
@@ -5,14 +5,17 @@ def get_oncall_email(client, service: dict) -> str:
     """Fetches the oncall's email for a given service."""
     escalation_policy_id = service["escalation_policy"]["id"]
     escalation_policy = client.rget(f"/escalation_policies/{escalation_policy_id}")
-    schedule_id = escalation_policy["escalation_rules"][0]["targets"][0]["id"]
+    filter_name = "{}_ids[]".format(
+        escalation_policy["escalation_rules"][0]["targets"][0]["type"].split("_")[0]
+    )
```

Comment:
Let's use f-strings vs format
```suggestion
    filter_name = (
        f"{escalation_policy['escalation_rules'][0]['targets'][0]['type'].split('_')[0]}_ids[]"
    )
```